be/src/exprs/expr_context.cpp

```diff
@@ -185,8 +185,8 @@ StatusOr<ColumnPtr> ExprContext::evaluate(vectorized::Chunk* chunk) {
 }
 
 StatusOr<ColumnPtr> ExprContext::evaluate(Expr* e, vectorized::Chunk* chunk) {
-    DCHECK(_prepared);
-    DCHECK(_opened);
+    // DCHECK(_prepared);
```

Comment:
？

Reply:
it was used in regression. I will remove it